### ISSUE TITLE
Ability to seed Scanner user/pass and Enforcer enforcerToken via Secrets / Helm values

### ISSUE THREAD

**Issue by agilgur5** [aquasecurity/aqua-helm#120]
Currently there are at least two _manual_ steps required in order to set-up all the Aqua components, meaning it is not fully automated and requires human intervention to complete.

Once the Server is installed, one must add the license key and admin user/pass. This actually [can be seeded with Helm or via secrets](https://github.com/aquasecurity/aqua-helm/blob/525c1aae5bfd31b8137fccbbe7353c134163fb14/server/values.yaml#L19), it's just not documented (which is a separate issue).

In order to fully automate installation, one must also be able seed a Scanner user/pass and an Enforcer group/token in the same way.
Currently one can only create a Scanner user/pass by manually going into the UI and creating one.
The same is true for an Enforcer group/token, you must do so in the UI.

Instead, or alternatively, one should also be able to specify those via k8s Secrets or Helm values, like one can for the admin user/pass and license key already. If that were possible, then one would be able to install the Scanner and the Enforcer without human intervention and make for a very seamless automated install.
I thought that must be possible when I first went about installing Aqua, after all automation is key and this whole repo is to help with automation, but I was proven quite wrong unfortunately 😕 

I'm not sure if this would require upstream changes in the Server, or if it is already possible to do so and the capabilities just need to be added to the Charts.

## Tags

Related to #119 , which points out the lack of docs for these manual steps.

**Comment by KoppulaRajender** [comment]
Added support for both scenarios, thankyou.

**Comment by michaelrdolan** [comment]
I was eagerly looking forward to this fix as well for the same automation reasons mentioned by @agilgur5.  It looks like there is a way to seed the scanner user/pass, but not a way to have the Enforcer token seeded.  Happy to be wrong about this, though.

Is there a way to seed the Enforcer token on the Server?

**Comment by agilgur5** [comment]
> Is there a way to seed the Enforcer token on the Server?

As far as I know, there is still no method of seeding the Enforcer `enforcerToken`.

In fact, the PRs that "fixed" this issue, i.e., #270 et al, actually _don't_ fix the issue for the Scanner user/pass either.

It added support for specifying the Scanner password via Secret, which fixed #118, but _did not fix_ this issue. The [docs added](https://github.com/aquasecurity/aqua-helm/pull/270/files#diff-26f6432cd17508fa13d1fff404ca4e21347f79e1987aab64a1a12071409fe2f3R29) in #270 even say a Scanner user/pass still has to be created via the UI. The chart changes there also do not affect the Server, which is what has to be seeded with the Scanner User and Enforcer, not the Scanner.
Notably, I did mention in my opening issue that support for this may "require upstream changes in the Server" itself if that capability did not exist. Given how much trouble I had with Aqua back when I made these issues (enough that we dropped them), if I had to guess, I would say they don't yet exist.

I believe @KoppulaRajender misinterpreted this issue as a duplicate of #118, which it is not, and so this should be re-opened, as one still cannot seed either as far as I can tell. Correct me if I'm wrong, but even in the current version I couldn't find a way to do this in the [Helm values](https://github.com/aquasecurity/aqua-helm/blob/f08fee61b097e6bd6bb571900f53af50b5ea2723/server/values.yaml) or README for the Server.
